packages/server-core/src/entities/component/component.seed.ts

```diff
@@ -169,6 +170,26 @@ export const componentSeed = {
       updatedAt: "2020-11-12 13:14:45",
       type: "postprocessing",
       entityId: entitySeed.templates.find(template => template.id === '1B698484-C15A-4CEC-9247-03873520DF70').id
-    }
+    },
+    {
+      id: "09cc03c4-24e9-sd1b-bc2e-e7e742fb069f",
+      data: {
+        options: {
+          probePosition:new Vector3(0),
```

Comment:
same here, use { x:0, ... }  instead of Vector3

Reply:
ok